test/deploy.sh

```diff
@@ -131,8 +135,23 @@ $DRYRUN $ROOT/bin/beacon -numShards $SHARDS > $log_folder/beacon.log 2>&1 | tee
 sleep 1 #waiting for beaconchain
 MA=$(grep "Beacon Chain Started" $log_folder/beacon.log | awk -F: ' { print $2 } ')
 
+echo "launching boot node ..."
+$DRYRUN $ROOT/bin/bootnode > $log_folder/bootnode.log 2>&1 | tee -a $LOG_FILE &
+sleep 1
+BN_MA=$(grep "BN_MA" $log_folder/bootnode.log | awk -F\= ' { print $2 } ')
```

Comment:
This type of information valuable to outside users should be exposed via a more suitable means than just log extraction.  Log parsing shouldn't be part of a deployment pipeline.  I'd say write a one-liner file and then read from the script.  Like a PID file.

Reply:
agree. this is an ugly hack. I have opened an issue for a better design.
https://github.com/harmony-one/harmony/issues/396